src/gevent_server.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf8 -*-
+from __future__ import unicode_literals
+from server import response_error
+from server import response_ok
+from server import resolve_uri
+from server import parse_request
```

Comment:
great work importing these from `server.py` where they already exist.  No need to duplicate that work.